Makefile

```diff
@@ -5,7 +5,5 @@ IMAGE_NAME ?= codeclimate/codeclimate-coffeelint
 image:
 	docker build --tag $(IMAGE_NAME) .
 
-citest:
-	docker run --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && bundle exec rake"
-
-test: image citest
+test: image
```

Comment:
please, can you update the first line `.PHONY`?

Reply:
Done!